src/types/timestamp.rs

```diff
@@ -1164,12 +1126,12 @@ impl From<ion_c_sys::timestamp::IonDateTime> for Timestamp {
             TSPrecision::Fractional(_) => Precision::Second,
         };
         let fractional_seconds = if let TSPrecision::Fractional(mantissa) = ionc_dt.precision() {
-            Some(match mantissa {
+            match mantissa {
                 IonCMantissa::Digits(digits) => Mantissa::Digits(*digits),
                 IonCMantissa::Fraction(fraction) => Mantissa::Arbitrary(fraction.clone().into()),
-            })
+            }
         } else {
-            None
+            Mantissa::Digits(0)
```

Comment:
I'm seeing `Mantissa::Digits(0)` in enough places that I think it would be worth implementing [the `Default` trait](https://doc.rust-lang.org/std/default/trait.Default.html) for `Mantissa`. What do you think?

Reply:
Agreed